components/builder-api-client/src/builder.rs

```diff
@@ -464,7 +466,7 @@ impl BuilderAPIClient {
                           .get(&format!("depot/origins/{}/encryption_key", origin)),
                       dst_path.as_ref(),
                       Some(token),
-                      Permissions::Standard,
+                      DEFAULT_PUBLIC_KEY_PERMISSIONS,
```

Comment:
Is this one right?

Reply:
It seems like this would be fine. Previously, the permissions would have been set to whatever the low-level calls set them to by default, but now they'd be `444` (on Linux, anyway; unchanged on Windows).

Reply:
Yeah, so my main question here really is: What is encryption_key? Is it a private or public bit of information?  If private, we likely don't want read for other.

Reply:
Based on the function's name, it looks like public information, so I think we're good here.